### DISCUSSION TITLE
Stage Loader / Fallback ?

### DISCUSSION THREAD

**Discussion by helloindra** [konvajs/konva#1310] [Q&A]
Hey Gangs,

Anyone know how to create loader/fallback for Konva Stage ?
I am trying to load many images to the stage, but i don't like to see the image loading 1-by-1.

I would like to try to create loader/fallback till the image were all loaded.

Many Thanks!

**Comment by lavrton** [comment]
What did you try? You can show a spinner, like rotating arc while the image is loading.

**Reply by helloindra** [reply]
Yes, i wanted to create that kind of things.
like this one.

https://codepen.io/desandro/pen/XWzqXrP

**Reply by lavrton** [reply]
Good. So, what did you try? You can do the same with Konva.
